indium-chrome.el

```diff
@@ -113,6 +113,21 @@ Try a maximum of NUM-TRIES."
           (port (read-from-minibuffer "Port: " (number-to-string indium-chrome-port))))
       (indium-chrome--get-tabs-data host port #'indium-chrome--connect-to-tab))))
 
+;;;###autoload
+(defun indium-connect-to-local-chrome-to-url (url)
```

Comment:
What about making the `HOST` and `PORT` parameters optional in `indium-chrome-connect-to-chrome-url` instead, defaulting to localhost:9229?